providers/mlx4/verbs.c

```diff
@@ -1328,6 +1334,10 @@ struct ibv_ah *mlx4_create_ah(struct ibv_pd *pd, struct ibv_ah_attr *attr)
 	if (query_port_cache(pd->context, attr->port_num, &port_attr))
 		return NULL;
 
+	if (port_attr.flags & IBV_QPF_GRH_REQUIRED &&
+	    !attr->is_global)
```

Comment:
Same question here.. Is there some expected future use for this in mlx4? This at least makes more sense since it will be set for roce ports..

Reply:
mlx4 turn on this bit today for the roce case, so it's applicable.